backend/app/literature/schemas/reference_category_schemas.py

```diff
@@ -14,3 +14,4 @@ class ReferenceCategory(str, Enum):
     Internal_Process_Reference = "internal_process_reference"
     Unknown = "unknown"
     Retraction = "retraction"
+    Test = "test"
```

Comment:
Hi @FB-ping  This probably shouldn't be committed.

Reply:
@azurebrd  I didn't touch this file.

Reply:
Oh, that's weird, thanks Pinglei.  Before we merge, we should roll it back.

Reply:
[5913f96](https://github.com/alliance-genome/agr_literature_service/pull/83/commits/5913f96ec90687d56d90c63a0992a3d3d5dce1a2) fixes that